xbmc/rendering/dx/DeviceResources.cpp

```diff
@@ -1220,6 +1224,7 @@ HDR_STATUS DX::DeviceResources::ToggleHDR()
     m_swapChain = nullptr;
     m_deferrContext->Flush();
     m_d3dContext->Flush();
+    m_IsTransferPQ = false;
```

Comment:
Added this line of code to fix manual HDR on/off switching with F11 key: GUI could stay indefinitely mapped to HDR when toggle HDR off in middle of playback.